spharpy/spherical.py

```diff
@@ -248,17 +249,24 @@ def n3d_to_sn3d_norm(n):
     return 1 / np.sqrt(2 * n + 1)
 
 
-def spherical_harmonic_basis(n_max, coords):
+def spherical_harmonic_basis(
+        n_max, coordinates, normalization="n3d", channel_convention="acn",
+        phase_convention='Condon-Shortley'):
     r"""
     Calculates the complex valued spherical harmonic basis matrix.
-
-    The spherical harmonic functions are fully normalized (N3D) and include the
-    Condon-Shotley phase term :math:`(-1)^m` [#]_, [#]_.
+    See also :func:`spherical_harmonic_basis_real`.
```

Comment:
```suggestion

    See [#]_ and [#]_ for details.
    See also :py:func:`spherical_harmonic_basis_real`.
```
we need to refer the references, otherwise the doc build on ci will fail